File: java/direct-policy/src/main/java/org/nhindirect/policy/PolicyRequiredException.java
package org.nhindirect.policy;

public class PolicyRequiredException extends PolicyProcessException
{
	static final long serialVersionUID = 492724799349319556L;
	
	   /////CLOVER:OFF
		/**
		 * {@inheritDoc}
		 */
	    public PolicyRequiredException() 
	    {
	    	super();
	    }

		/**
		 * {@inheritDoc}
		 */
	    public PolicyRequiredException(String msg) 
	    {
	        super(msg);
	    }

		/**
		 * {@inheritDoc}
		 */
	    public PolicyRequiredException(String msg, Throwable t) 
	    {
	        super(msg, t);
	    }

		/**
		 * {@inheritDoc}
		 */
	    public PolicyRequiredException(Throwable t) 
	    {
	        super(t);
	    }
	    /////CLOVER:ON
}

File: csharp/policy/PolicyRequiredException.cs
using System;

namespace Health.Direct.Policy
{
    public class PolicyRequiredException : PolicyProcessException
    {
        public PolicyRequiredException() { }


        public PolicyRequiredException(String msg) : base(msg) { }

        public PolicyRequiredException(String msg, Exception ex) : base(msg, ex) { }

    }
}
